woocommerce/woocommerce-gateway-stripe
# Bump L-2 versions for WP and WC

Fixes #

## Changes proposed in this Pull Request:

This PR bumps the supported versions of WordPress and WooCommerce to their latest - 2 versions:

- PHP: `7.3`, `7.4` and `8.0`
- WordPress: `6.0`, `5.9` and `5.8`
- WooCommerce: `6.8.2`, `6.9.4` and `7.0.0`



**Notes:**
- "phpunit/phpunit": "7.5.20" is still needed by WordPress 5.8

## Testing instructions
Verify that ALL the `checks` for the PR are successfully completed

![Screenshot 2022-11-01 at 13 42 24](https://user-images.githubusercontent.com/407542/199288473-e3fc8745-c186-4537-8e0a-4ce046ce8b77.png)

---


**Post merge**
